bhop/bhop_internal.inc

```diff
@@ -97,6 +167,15 @@ PITCH_EFFECT_NOTE_DOWN    = %00010000
 PITCH_EFFECT_ARP          = %00100000
 PITCH_EFFECT_TRIGGERED    = %10000000
 
+; expansion flags, for multichip handling
+EXPANSION_VRC6 = %00000001
+EXPANSION_VRC7 = %00000010
+EXPANSION_FDS  = %00000100
+EXPANSION_MMC5 = %00001000
+EXPANSION_N163 = %00010000
+EXPANSION_S5B  = %00100000
+EXPANSION_VT02 = %01000000  ; are we gonna support this?
```

Comment:
I have no immediate plans, but if it's part of the spec the flag should be there, if only so we remember not to allocate it to something else in the future.